### PR TITLE
Add sros show lag port

### PR DESCRIPTION
**Pull request by bennnnnnnn**
Add the show lag port command for Alcatel SROS

### CODE REVIEW

File: ntc_templates/templates/alcatel_sros_show_lag_port.textfsm [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+Start
+  ^----------- -> LagPort
+
+LagPort
+  ^${LAG_ID}\s+${PORT_ID}\s+${ADM}\s+${ACT_STD}\s+${OPR}\s+${PRIMARY}\s+${SUB_GROUP}\s+${FORCED}\s+${PRIO}(\s|$$) -> Record
+  ^\s+${PORT_ID}\s+${ADM}\s+${ACT_STD}\s+${OPR}\s+${SUB_GROUP}\s+${FORCED}\s+${PRIO}(\s|$$) -> Record
+  ^\s*$$
+  ^-----------
+  ^===========
```
Comment (mjbear) [inline ntc_templates/templates/alcatel_sros_show_lag_port.textfsm:19]:
```suggestion
  ^-+
  ^=+
```

Reduce characters in the horizontal separator lines by using regex with a plus sign

File: ntc_templates/templates/alcatel_sros_show_lag_port.textfsm [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+Value ADM (up|down)
+Value ACT_STD (active|standby)
+Value OPR (up|down)
+Value PRIMARY (yes|no)
+Value SUB_GROUP (\d+)
+Value FORCED (-)
+Value PRIO (\d+)
+
+Start
+  ^----------- -> LagPort
```
Comment (mjbear) [inline ntc_templates/templates/alcatel_sros_show_lag_port.textfsm:12]:
```suggestion
Start
  ^-+ -> LagPort
  ^=+
  ^Lag\s+Port\s+States
  ^LACP\s+Status:
  ^Lag-id\s+
  ^\s+Stdby\s*
  ^\s*$$
  ^. -> Error
```

I'd reduce the characters for the horizontal separators by using a regex with a plus sign.

Also add in the extra lines (before the state transition to `LagPort`) and `Error` rule for the `Start` state.

Comment (mjbear) [reply ntc_templates/templates/alcatel_sros_show_lag_port.textfsm:12]: Not looking to rag on you -- simply inform.
https://github.com/networktocode/ntc-templates/blob/master/docs/user/faq.md#why-is-there-a-requirement-to-use-error-in-every-template
https://github.com/networktocode/ntc-templates/blob/master/docs/dev/dev_parser.md#error-directive

Precedent template with `Error` in each state.
https://github.com/networktocode/ntc-templates/blob/master/ntc_templates/templates/alcatel_sros_show_router_mpls_interface.textfsm

File: ntc_templates/templates/index [hunk 3]
```diff
@@ -30,6 +30,7 @@ alcatel_sros_show_router_mpls_lsp.textfsm, .*, alcatel_sros, sh[[ow]] router mpl
 alcatel_sros_show_service_id_base.textfsm, .*, alcatel_sros, sh[[ow]] serv[[ice]] id ba[[se]]
 alcatel_sros_show_service_sdp.textfsm, .*, alcatel_sros, sh[[ow]] service sdp
 alcatel_sros_show_system_cpu.textfsm, .*, alcatel_sros, sh[[ow]] system cpu
+alcatel_sros_show_lag_port.textfsm, .*, alcatel_sros, sh[[ow]] lag (\d+ )?port
```
Comment (mjbear) [inline ntc_templates/templates/index:33]: @bennnnnnnn
I don't have any Alcatel/Nokia SROS gear to poke at so I'm going to pose questions below.

After the `show` word, does `lag` always need spelled out fully? Or is there a shortcut of say `la`.

Similarly for `port` after the interface number, from the CLI is it possible to issue a shorter keyword?
(If yes `port` could be changed to accept abbreviations.)

**Review by mjbear — CHANGES_REQUESTED** [review]

File: ntc_templates/templates/alcatel_sros_show_lag_port.textfsm [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+LagPort
+  ^${LAG_ID}\s+${PORT_ID}\s+${ADM}\s+${ACT_STD}\s+${OPR}\s+${PRIMARY}\s+${SUB_GROUP}\s+${FORCED}\s+${PRIO}(\s|$$) -> Record
+  ^\s+${PORT_ID}\s+${ADM}\s+${ACT_STD}\s+${OPR}\s+${SUB_GROUP}\s+${FORCED}\s+${PRIO}(\s|$$) -> Record
+  ^\s*$$
+  ^-----------
+  ^===========
+  ^Total
+  ^. -> Error
```
Comment (jmcgill298) [inline ntc_templates/templates/alcatel_sros_show_lag_port.textfsm:28]:
```suggestion
  ^. -> Error

```

File: ntc_templates/templates/alcatel_sros_show_lag_port.textfsm (hunk 2 above)
Comment (bennnnnnnn) [reply ntc_templates/templates/alcatel_sros_show_lag_port.textfsm]: Thanks, was not aware! I've incorporated these changes 👍

File: ntc_templates/templates/alcatel_sros_show_lag_port.textfsm [hunk 5]
```diff
@@ -0,0 +1,28 @@
+Value LAG_ID (\d+\(\w\))
+Value PORT_ID (\d+/\d+/\d+)
+Value ADM (up|down)
+Value ACT_STD (active|standby)
+Value OPR (up|down)
```
Comment (jmcgill298) [inline ntc_templates/templates/alcatel_sros_show_lag_port.textfsm:5]:
```suggestion
Value ADMIN_STATUS (up|down)
Value ACTIVE_STATUS (active|standby)
Value OPERATIONAL_STATUS (up|down)
```

File: ntc_templates/templates/alcatel_sros_show_lag_port.textfsm [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+Value PRIMARY (yes|no)
+Value SUB_GROUP (\d+)
+Value FORCED (-)
+Value PRIO (\d+)
```
Comment (jmcgill298) [inline ntc_templates/templates/alcatel_sros_show_lag_port.textfsm:9]:
```suggestion
Value PRIORITY (\d+)
```

File: ntc_templates/templates/alcatel_sros_show_lag_port.textfsm (hunk 1 above)
Comment (bennnnnnnn) [reply ntc_templates/templates/alcatel_sros_show_lag_port.textfsm]: Fixed!

File: ntc_templates/templates/alcatel_sros_show_lag_port.textfsm [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+  ^Lag\s+Port\s+States
+  ^LACP\s+Status:
+  ^Lag-id\s+
+  ^\s+Stdby\s*
+  ^\s*$$
+  ^. -> Error
+
+LagPort
+  ^${LAG_ID}\s+${PORT_ID}\s+${ADM}\s+${ACT_STD}\s+${OPR}\s+${PRIMARY}\s+${SUB_GROUP}\s+${FORCED}\s+${PRIO}(\s|$$) -> Record
+  ^\s+${PORT_ID}\s+${ADM}\s+${ACT_STD}\s+${OPR}\s+${SUB_GROUP}\s+${FORCED}\s+${PRIO}(\s|$$) -> Record
```
Comment (jmcgill298) [inline ntc_templates/templates/alcatel_sros_show_lag_port.textfsm:23]:
```suggestion
  ^${LAG_ID}\s+${PORT_ID}\s+${ADMIN_STATUS}\s+${ACTIVE_STATUS}\s+${OPERATIONAL_STATUS}\s+${PRIMARY}\s+${SUB_GROUP}\s+${FORCED}\s+${PRIORITY}(\s|$$) -> Record
  ^\s+${PORT_ID}\s+${ADMIN_STATUS}\s+${ACTIVE_STATUS}\s+${OPR}\s+${SUB_GROUP}\s+${FORCED}\s+${PRIORITY}(\s|$$) -> Record
```

File: tests/alcatel_sros/show_lag_port/show_lag_port.yml [hunk 8]
```diff
@@ -0,0 +1,164 @@
...
+parsed_sample:
+  - act_std: "active"
+    adm: "up"
+    forced: "-"
+    lag_id: "5(d)"
+    opr: "down"
+    port_id: "1/1/15"
+    primary: "yes"
+    prio: "32768"
+    sub_group: "1"
```
Comment (jmcgill298) [inline tests/alcatel_sros/show_lag_port/show_lag_port.yml:11]:
```suggestion
  - active_status: "active"
    admin_status: "up"
    forced: "-"
    lag_id: "5(d)"
    operational_status: "down"
    port_id: "1/1/15"
    primary: "yes"
    priority: "32768"
    sub_group: "1"
```

**Review by jmcgill298 — APPROVED** [review]

**Pull request merged**